automation/src/test/java/MakeModel.java

```diff
@@ -0,0 +1,27 @@
+public class MakeModel {
+
+	private String make;
+	private String model;
+
+	public String getMake() {
+		return make;
+	}
+
+	public void setMake(String make) {
+		this.make = make;
+	}
+
+	public String getModel() {
+		return model;
+	}
+
+	public void setModel(String model) {
+		this.model = model;
+	}
+
+	@Override
+	public String toString() {
```

Comment:
If the intention of this is to print all fields know that you'll have to maintain it if any fields are added to the class. Sort of breaks Open Close principle.